nestedtensor/csrc/python_functions.cpp

```diff
@@ -32,6 +33,9 @@ at::Tensor cross_entropy(
   if (ignore_index.has_value()) {
     options = options.ignore_index(ignore_index.value());
   }
+  if (label_smoothing.has_value()) {
+    options = options.label_smoothing(label_smoothing.value());
```

Comment:
Check failures are due to `F::CrossEntropyFuncOptions` not having a `label_smoothing` arg, although it's present in master (https://github.com/pytorch/pytorch/blob/92b31b59aff908fcecb76e5baff5bb52ce62608a/torch/csrc/api/include/torch/nn/options/loss.h#L666). What's the deal here?